proto/greptime/v1/region/server.proto

```diff
@@ -106,7 +106,10 @@ message CreateRequest {
 // Same as CreateRequests, but for dropping regions.
 message DropRequests { repeated DropRequest requests = 1; }
 
-message DropRequest { uint64 region_id = 1; }
+message DropRequest { 
+  uint64 region_id = 1;
+  bool fast_drop_database_path = 2;
```

Comment:
How about using `fast_drop_path`? I believe we might also offer the ability to fast drop physical tables in the future, so it's not just for dropping databases.

Reply:
of course will change it

Reply:
Done